core/runtime/jsruntime.rs

```diff
@@ -907,59 +921,89 @@ impl JsRuntime {
     module_map.mod_evaluate_sync(scope, mod_id).unwrap();
   }
 
+  /// Executes built-in scripts and ES modules.
```

Comment:
Update the comment to explain why this function is special - essentially this code is required for op system and everything else to function properly. Explain why it's not done in `bindings::initialize_primordials_and_infra`